addons/mail/models/mail_mail.py

```diff
@@ -40,9 +41,16 @@ def default_get(self, fields):
 
     # content
     mail_message_id = fields.Many2one('mail.message', 'Message', required=True, ondelete='cascade', index=True, auto_join=True)
+
+    # Mail Message fields
+    message_int = fields.Integer(compute='_compute_message_int', compute_sudo=True)
```

Comment:
``mail_message_id_int`` maybe ? We already have that kind of fields (``mail_mail_id_int``, ``sms_sms_id_int``, ...) that way we keep the prefixes :)